deployment/common/proposalutils/propose.go

```diff
@@ -39,6 +39,37 @@ func BuildProposalMetadata(
 	return metaDataPerChain, nil
 }
 
+// BuildProposalMetadataWithOffsets builds the metadata for a proposa;, offsetting the startingOpCount.
+func BuildProposalMetadataWithOffsets(
```

Comment:
Don't think changesets would ever call thing func directly right? If so can be made package-private.

Reply:
https://github.com/smartcontractkit/chainlink/pull/15877/commits/afc13ca92a4c18ced03712680bcab6df42c0f512